olp-cpp-sdk-core/include/olp/core/client/OlpClientSettings.h

```diff
@@ -222,6 +253,11 @@ struct CORE_API OlpClientSettings {
    */
   RetrySettings retry_settings;
 
+  /**
+   * @brief API Lookup settings.
+   */
+  ApiLookupSettings api_lookup_settings;
```

Comment:
This and the above ApiLookupSettings are not in the scope of this ticket, please remove.